### PR TITLE
Feature/ATG-59 class Block

### PR DESCRIPTION
**Pull request by IraCherginyshka**
Create class Block

### CODE REVIEW

File: src/app/models/Block.model.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import { FIGURE_SIZE, BORDER_COLOR } from '../constants/board-component-constants';
```
Comment (oleksandrstarov) [inline src/app/models/Block.model.ts]: Please rename file to start with lower case

File: src/app/models/Block.model.ts [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+export default class BlockModel {
```
Comment (oleksandrstarov) [inline src/app/models/Block.model.ts]: No need to use default as we using import with destructing assignment import { BlockModel } from ...

File: src/app/models/Block.model.ts [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+  constructor(private ctx: CanvasRenderingContext2D, color: string) {
+    this.ctx.fillStyle = color;
+    this.ctx.strokeStyle = BORDER_COLOR;
+  }
+
+  fillBoardBlock(x: number, y: number) {
```
Comment (oleksandrstarov) [inline src/app/models/Block.model.ts]: please specify return type

File: src/app/models/Block.model.ts [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+    this.ctx.fillRect(x * FIGURE_SIZE, y * FIGURE_SIZE, FIGURE_SIZE, FIGURE_SIZE);
```
Comment (oleksandrstarov) [inline src/app/models/Block.model.ts]: The FIGURE_SIZE is ambiguous - we have 7 figures in tetris. Which one do we using here?

**Review by oleksandrstarov — CHANGES_REQUESTED** [review]

File: src/app/models/Block.model.ts (hunk 1 above)
Comment (IraCherginyshka) [reply src/app/models/Block.model.ts]: Done

File: src/app/models/Block.model.ts (hunk 2 above)
Comment (IraCherginyshka) [reply src/app/models/Block.model.ts]: Done

File: src/app/models/Block.model.ts (hunk 4 above)
Comment (IraCherginyshka) [reply src/app/models/Block.model.ts]: FIGURE_SIZE it is actually BLOCK_SIZE. I'll rename this constant

File: src/app/models/Block.model.ts (hunk 3 above)
Comment (IraCherginyshka) [reply src/app/models/Block.model.ts]: Done

**Review by oleksandrstarov — APPROVED** [review]

**Comment by oleksandrstarov** [comment]
Please, resolve conflicts

**Pull request merged**